server_code/_nucypher.py

```diff
@@ -1,4 +1,4 @@
-from secrets import key
+from secrets import Enricokey
```

Comment:
Given that `EnricoKey` is a public key, it's in the wrong namespace (not secret)?